po/fr.po

```diff
@@ -166,49 +164,46 @@ msgstr ""
 
 #: newsboat.cpp:177
 msgid "It bundles:"
-msgstr ""
+msgstr "Inclut:"
 
 #: newsboat.cpp:178
-#, fuzzy
 msgid ""
 "- JSON for Modern C++ library, licensed under the MIT License: https://"
 "github.com/nlohmann/json"
 msgstr ""
-"Il contient la bibliothèque JSON for Modern C++, publiée sous la licence MIT/"
-"X Consortiumhttps://github.com/nlohmann/json"
+"- la bibliothèque JSON for Modern C++, publiée sous la licence MIT/"
+"X Consortium https://github.com/nlohmann/json"
 
 #: newsboat.cpp:181
-#, fuzzy
 msgid ""
 "- optional-lite library, licensed under the Boost Software License: https://"
 "github.com/martinmoene/optional-lite"
 msgstr ""
-"Il contient la bibliothèque JSON for Modern C++, publiée sous la licence MIT/"
-"X Consortiumhttps://github.com/nlohmann/json"
+"- la bibliothèque optional-lite, publiée sous la licence Boost Software:"
+"https://github.com/martinmoene/optional-lite"
 
 #: newsboat.cpp:184
-#, fuzzy
 msgid ""
 "- expected-lite library, licensed under the Boost Software License: https://"
 "github.com/martinmoene/expected-lite"
 msgstr ""
-"Il contient la bibliothèque JSON for Modern C++, publiée sous la licence MIT/"
-"X Consortiumhttps://github.com/nlohmann/json"
+"- la bibliothèque expected-lite, publiée sous la licence Boost Software:"
+"https:github.com/martinmoene/expected-lite"
```

Comment:
Same, a space between colon and URL. The double slashes in the URL went missing, too.